config/settings.yml

```diff
@@ -2,4 +2,4 @@ hold_button_url: http://cat.libraries.psu.edu/cgi-bin/catredirpg?C=
 matomo_id: 10
 solr:
   url: http://127.0.0.1:8983
-  collection: psul_blacklight
\ No newline at end of file
+  collection: psul_catalog
```

Comment:
yup better name ✔️

Reply:
yeah figured I'd change this while I was at it